linkerd/docs/config.md

```diff
@@ -144,6 +145,36 @@ This administrative interface was originally based on
 be found at
 [TwitterServer HTTP Admin interface](https://twitter.github.io/twitter-server/Admin.html).
 
+#### Admin security
+
+By default all administrative endpoints are accessible.
+This might not always be desired as this allows for example everybody to shutdown the process if the administrative port is not closed because metrics should be polled from Linkerd.
+The admin security configuration allows to whitelist or blacklist dedicated administrative endpoints.
+
+```yaml
+admin:
+  ip: 127.0.0.1
+  port: 9990
+  security:
+    uiEnabled: true
+    controlEnabled: false
+    pathWhitelist:
+    - ^/logging[.]json$
+```
+
+The security configuration supports these parameters allowing fine grained control over the administrative endpoints that are available.
+
+Key | Default Value | Description
+--- | ------------- | -----------
+uiEnabled | true | Configures if all endpoints required to run the administrative ui are available
+controlEnabled | true | Configures if the endpoints to control Linkerd are available, i.e. `/admin/shutdown` and `/logging.json`
+diagnosticsEnabled | true | Configures if all endpoints are available.
```

Comment:
As described here, I think setting "diagnosticsEnabled: true" and "controlEnabled: false" would still allow access to the control endpoints, which doesn't seem like the right behavior.  I think diagnostics should be everything else, not included in ui or control.

What do you think about this algorithm:
1. determine which category a request is for: ui, control, or diagnostics
2. if that category is enabled and the path doesn't match the blacklist, the request is allowed
3. if that category is disabled but the path matches the whitelist and not the blacklist, the request is allowed
4. otherwise the request is rejected

Reply:
Oh yes, that’s a good point. 
What do you think about adding the ui and control endpoints to the blacklist if the corresponding parameter is set to false?

I am unsure atm how to find out the category.

Reply:
I’ll try to implement this tomorrow when I have a fresh mind :)